src/components/global/Playground/index.tsx

```diff
@@ -92,13 +92,15 @@ export default function Playground({
   src,
   size = 'small',
   devicePreview,
+  defaultCodeExpanded = false,
 }: {
   code: { [key in UsageTarget]?: MdxContent | UsageTargetOptions };
   title?: string;
   src: string;
   size: string;
   description?: string;
   devicePreview?: boolean;
+  defaultCodeExpanded: boolean
```

Comment:
How about `expandCodeByDefault`? Too wordy? I want to make it clear that it's only used as the default, rather than anything tied to the active state.

Reply:
Works for me 👍